app_legacy/Helpers/render/content.php

```diff
@@ -26,7 +26,7 @@ function RenderActivePlayersComponent(): void
                         <tr>
                             <td data-bind='html: playerHtml'></td>
                             <td data-bind='html: gameHtml'></td>
-                            <td data-bind='text: richPresence' class="w-full"></td>
+                            <td data-bind='text: richPresence' class="w-full break-words"></td>
```

Comment:
Similar to https://github.com/RetroAchievements/RAWeb/pull/1368#discussion_r1108776953, I had to use `break-all` to make this work in Chrome (did not test Edge or Firefox)

Reply:
In this case, it might have been my choice of test data.
![image](https://user-images.githubusercontent.com/32680403/219458868-d68b76d4-a380-409b-90a7-d69ac076029e.png)

Reply:
Assuming it's working here for you on Chrome, it's interesting that it's an issue on another page. Can you confirm `break-words` is functioning correctly for you here?

Reply:
It is not.
> I had to use `break-all` to make this work in Chrome

Reply:
Thank you for the confirmation, I got confused by the follow-up comment. I will make the same adjustment to this PR as I make in the other one later today.

Reply:
I can now repro what you're seeing. I think this was a problem with Tailwind & Vite on my end. Fixed.